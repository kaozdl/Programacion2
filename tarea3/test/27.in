# kesimo_abb
insertar_binario (15,q) .
kesimo_subarbol 1 .
insertar_binario (12,d) .
insertar_binario (18,o) (21,v) .
insertar_binario (30,t) .
kesimo_subarbol 1 .
kesimo_subarbol 2 .
kesimo_subarbol 3 .
kesimo_subarbol 4 .
kesimo_subarbol 5 .
Fin
