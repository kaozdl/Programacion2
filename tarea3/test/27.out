1>
2>
(15,q)
0
0
3>(15,q)
4>
(15,q)
-(12,d)
1
0
5>
-(21,v)
(15,q)
--(18,o)
-(12,d)
2
1
6>
-(21,v)
--(30,t)
(15,q)
--(18,o)
-(12,d)
2
2
7>(12,d)
8>(18,o)
9>(15,q)
10>(30,t)
11>(21,v)
12>Fin.
